filebeat/reader/multiline/multiline_test.go

```diff
@@ -150,6 +151,25 @@ func TestMultilineBeforeNegateOKWithEmptyLine(t *testing.T) {
 	)
 }
 
+func TestMultilineAfterTruncated(t *testing.T) {
+	pattern := match.MustCompile(`^[ ]`) // next line is indented a space
+	maxLines := 2
+	testMultilineTruncated(t,
```

Comment:
Could you also add a test where the truncated flag should be misssing?

Reply:
Done.